apps/SageAccountWeb/src/components/RegisterAccount1.tsx

```diff
@@ -148,36 +170,31 @@ export const RegisterAccount1 = () => {
         alias: username,
         type: AliasType.USER_NAME,
       })
-      if (!aliasCheckResponse.available) {
-        displayToast('Sorry, that username has already been taken.', 'danger')
-      } else if (!aliasCheckResponse.valid) {
-        displayToast('Sorry, that username is not valid.', 'danger')
```

Comment:
Need to check `valid` flag first. Extracted this check to a `validateAlias` utility.